legacy/reap/regionheap.h

```diff
@@ -47,7 +47,7 @@ using namespace HL;
 template <class Sbrk>
 class LeaHeap2 :
   public
-  Threshold<4096,
+  Threshold<4096, // FIXME: is this page-size dependent?
```

Comment:
@bpowers there were a couple of parts where I wasn't sure if the number was referring to the page size or another value, like here and below. I'm guessing this one should be changed to the page size (though not sure about the others), but do you know for sure?

Reply:
This particular heap layer is legacy code; I'm not sure it's being used by anyone. IIRC the size here was chosen arbitrarily (by me) but it might make sense to parameterize it by the system page size.